src/mixxx.cpp

```diff
@@ -449,6 +449,11 @@ void MixxxMainWindow::initialize(QApplication* pApp, const CmdlineArgs& args) {
             m_pLibrary,
             &Library::onSkinLoadFinished);
 
+    connect(this,
+            &MixxxMainWindow::newSkinLoaded,
```

Comment:
How about `skinLoaded`? And I think the slot should also be named `onSkinLoaded` or `slotSkinLoaded` respectively.

Reply:
The name of the signal already existed (it is connected on the previous sentence).
Should I still change it?